unity-renderer/Assets/DCLFeatures/CameraReel/ScreenshotViewer/Scripts/ScreenshotViewerView.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCLFeatures.CameraReel.ScreenshotViewer
+{
+    public class ScreenshotViewerView : MonoBehaviour
+    {
+        private const float SIDE_PANEL_ANIM_DURATION = 0.5f;
+
+        [SerializeField] private Image screenshotImage;
+        [SerializeField] private RectTransform rootContainer;
+
+        [Header("NAVIGATION BUTTONS")]
+        [SerializeField] private Button closeView;
+        [SerializeField] private Button prevScreenshotButton;
+        [SerializeField] private Button nextScreenshotButton;
+
+        private MetadataSidePanelAnimator metadataSidePanelAnimator;
+
+        private bool metadataPanelIsOpen = true;
+
+        [field: SerializeField] public ScreenshotViewerActionsPanelView ActionPanel { get; private set; }
+        [field: SerializeField] public ScreenshotViewerInfoSidePanelView InfoSidePanel { get; private set; }
+
+        public event Action CloseButtonClicked;
+        public event Action PrevScreenshotClicked;
+        public event Action NextScreenshotClicked;
+
+        private void Awake()
+        {
+            metadataSidePanelAnimator = new MetadataSidePanelAnimator(rootContainer, ActionPanel.InfoButtonBackground);
+        }
+
+        private void OnEnable()
+        {
+            closeView.onClick.AddListener(() => CloseButtonClicked?.Invoke());
+            prevScreenshotButton.onClick.AddListener(() => PrevScreenshotClicked?.Invoke());
+            nextScreenshotButton.onClick.AddListener(() => NextScreenshotClicked?.Invoke());
+        }
+
+        private void OnDisable()
+        {
+            closeView.onClick.RemoveAllListeners();
+            prevScreenshotButton.onClick.RemoveAllListeners();
+            nextScreenshotButton.onClick.RemoveAllListeners();
+        }
+
+        public void Hide()
+        {
+            gameObject.SetActive(false);
+        }
+
+        public void Show()
+        {
+            gameObject.SetActive(true);
```

Comment:
if it has a canvas setting the canvas alpha is better than setting the active state